types/index.d.ts

```diff
@@ -114,7 +114,11 @@ export interface VuetifyBreakpoint {
   xsOnly: boolean
 }
 
+export type RGB = number
+export type XYZ = [number, number, number]
+export type LAB = [number, number, number]
 export type VuetifyThemeItem = string | number
+export type VuetifyParsedThemeItem = RGB
```

Comment:
These are all internal, should be somewhere in utils instead. `VuetifyParsedTheme` is compatible with `VuetifyTheme` and doesn't need to exist.